projects/core/src/occ/config-loader/occ-config-loader.module.ts

```diff
@@ -27,6 +35,8 @@ export function initConfig(
 }
 
 /**
+ * @deprecated since 3.2, use SiteContextConfigLoaderModule instead
```

Comment:
Would you mind creating a ticket for 4.0 to get rid of all deprecated API in this PR?

Reply:
I will do it this week.

Reply:
Created https://github.com/SAP/spartacus/issues/11515